fv3core/stencils/tracer_2d_1l.py

```diff
@@ -34,23 +34,17 @@ def flux_y(cy: sd, dya: sd, dx: sd, sin_sg4: sd, sin_sg2: sd, yfx: sd):
         )
 
 
-@gtscript.function
-def mult_frac(var, frac):
-    var_tmp = var
-    return var_tmp * frac
-
-
 @gtstencil()
-def cmax_split_vars(
+def cmax_multiply_by_frac(
     cxd: sd, xfx: sd, mfxd: sd, cyd: sd, yfx: sd, mfyd: sd, frac: float
 ):
     with computation(PARALLEL), interval(...):
```

Comment:
I'll add this simple one for now. Documenting these inputs needs to be done along with some renames and generally an overhaul of this file.
```suggestion
def cmax_multiply_by_frac(
    cxd: sd, xfx: sd, mfxd: sd, cyd: sd, yfx: sd, mfyd: sd, frac: float
):
    """multiply all other inputs in-place by frac."""
    with computation(PARALLEL), interval(...):
```